### ISSUE TITLE
cloning an UpdateControl throws IllegalArgumentException

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#997]
Currently (in `master` branch at 3.3-6689) the `jme3test.terrain.TerrainGridSerializationTest` app crashes with the following stack trace:
```
Dec 31, 2018 10:35:55 AM com.jme3.material.plugins.J3MLoader readTechnique
WARNING: Fixed function technique was ignored
Dec 31, 2018 10:35:55 AM com.jme3.material.plugins.J3MLoader readTechnique
WARNING: Fixed function technique 'Default' was ignored for material Common/MatDefs/Terrain/HeightBasedTerrain.j3md
Dec 31, 2018 10:35:55 AM com.jme3.app.LegacyApplication handleError
SEVERE: Uncaught exception thrown in Thread[jME3 Main,5,main]
java.lang.IllegalArgumentException
	at java.util.concurrent.ConcurrentLinkedQueue.addAll(ConcurrentLinkedQueue.java:526)
	at com.jme3.scene.control.UpdateControl.jmeClone(UpdateControl.java:95)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:249)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:160)
	at com.jme3.util.clone.ListCloneFunction.cloneFields(ListCloneFunction.java:66)
	at com.jme3.util.clone.ListCloneFunction.cloneFields(ListCloneFunction.java:43)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:228)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:160)
	at com.jme3.scene.Spatial.cloneFields(Spatial.java:1446)
	at com.jme3.scene.Node.cloneFields(Node.java:725)
	at com.jme3.terrain.geomipmap.TerrainQuad.cloneFields(TerrainQuad.java:1801)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:255)
	at com.jme3.util.clone.Cloner.clone(Cloner.java:160)
	at com.jme3.scene.Spatial.clone(Spatial.java:1359)
	at com.jme3.scene.Node.clone(Node.java:684)
	at com.jme3.terrain.geomipmap.TerrainQuad.clone(TerrainQuad.java:1768)
	at com.jme3.terrain.geomipmap.TerrainQuad.clone(TerrainQuad.java:1763)
	at com.jme3.terrain.geomipmap.TerrainQuad.clone(TerrainQuad.java:109)
	at com.jme3.asset.CloneableAssetProcessor.createClone(CloneableAssetProcessor.java:48)
	at com.jme3.asset.DesktopAssetManager.registerAndCloneSmartAsset(DesktopAssetManager.java:317)
	at com.jme3.asset.DesktopAssetManager.loadAsset(DesktopAssetManager.java:379)
	at com.jme3.asset.DesktopAssetManager.loadModel(DesktopAssetManager.java:416)
	at com.jme3.asset.DesktopAssetManager.loadModel(DesktopAssetManager.java:420)
	at jme3test.terrain.TerrainGridSerializationTest.simpleInitApp(TerrainGridSerializationTest.java:50)
	at com.jme3.app.SimpleApplication.initialize(SimpleApplication.java:220)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.initInThread(LwjglAbstractDisplay.java:130)
	at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:211)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by stephengold** [comment]
Per the JavaDoc for `ConcurrentLinkedQueue`:
```
 Attempts to addAll of a queue to itself result in IllegalArgumentException.
```

**Comment by stephengold** [comment]
Fixed in `master` branch at hash fceee58546bf9eac988d3cd0cbbb2ab5d2ecc19c .

**Comment by stephengold** [comment]
See also https://github.com/jMonkeyEngine/sdk/issues/96
